cfgov/cfgov/settings/base.py

```diff
@@ -749,6 +752,10 @@
         # Boolean to turn it off explicitly unless enabled by another condition
         {'condition': 'boolean', 'value': False}
     ],
+
+    'ENABLE_WAGTAIL_API': [
+        {'condition': 'environment is not', 'value': 'production'},
```

Comment:
The data team's goal is to use the API in Staging, so my goal is to enable it there, as well as in dev, local, and test environments. 

We don't want it exposed to the public, i.e. in Prod or Beta. I wasn't sure how to accomplish that with flag conditions. Will this work? Should I add a line like this?:

```
{'condition': 'environment is not', 'value': 'beta'},
```